dali/kernels/slice/slice_flip_normalize_permute_pad_gpu.h

```diff
@@ -223,16 +223,21 @@ class SliceFlipNormalizePermutePadGpu {
       // We fuse the last dimension with the previous IF:
       // 1. There are at least 2 dimensions
       // 2. Last dimension is not sliced/padded/permuted
+      //    - if out_stride[last_dim] == abs(in_stride[last_dim]), then it is not permuted
+      //    - if anchor[last_dim] == 0 && out_shape[last_dim] == in_shape[last_dim],
+      //      then it is not sliced/padded
       // 3. Last dimension is not the channel dimension
-      // 4. The last two dimensions are either both flipped or not flipped
+      // 4. The in_strides of the last dimension and the former one align
+      //    - in_strides[last_dim] * in_shape[last_dim] == in_strides[last_dim - 1]
       int last_dim = Dims - 1;
       while (last_dim > 0 &&
              sample_desc.out_strides[last_dim] ==
                  static_cast<uint64_t>(sample_desc.in_strides[last_dim]) &&
```

Comment:
The comment above mentions `abs`, but it's not present in the code. If it's an intentional omission, then the comment should be updated.

Reply:
Done

Reply:
@jantonguirao In my head the abs should make sense there, but when I tried to put that in, test fail, and I was unable to debug it in a limited amount of time.

Let's say that out strides are [5, 1] and in strides are [-5, -1]. We should be able to compress it into out strides [1] and in strides [-1]. The new condition guarantees that the signs are correct.

This isn't a priority in any way, but I'm just raising the issue.